.github/workflows/updater-check.yml

```diff
@@ -0,0 +1,19 @@
+---
+on:
+  push:
+    branches:
+    - 'yli3/**'
+  workflow_dispatch: {}
+  schedule:
+    # Run weekly
+    - cron: '0 * * * *'
+
+jobs:
+  test_schedule:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.17.0'
+    - run: go run ./scanner/cmd/update.go
```

Comment:
You don't usually want to `go run` a single `.go` file

Reply:
Also, when running from the top-level directory, go will assume we are in the `github.com/stackrox/rox` module (defined in the `go.mod` in the top-level directory). We want to run things from Scanner's module. You should prefix this command with `cd scanner &&`, so it will use the Scanner module